src/components/SerialMonitor.vue

```diff
@@ -68,60 +80,42 @@ const portDisplayPacketsLimits = ref<Record<string, number>>({});
 const portNames = computed(() => Object.keys(app.packets));
 
 const selectedPort = computed(() => {
-    const selectedPortName = portNames.value[selectedPortIndex.value]
-    const selectedPort = app.managedSerialPorts.find(port => port.name === selectedPortName) || null;
+  const selectedPortName = portNames.value[selectedPortIndex.value];
+  const selectedPort =
+    app.managedSerialPorts.find((port) => port.name === selectedPortName) ||
+    null;
 
-    return selectedPort
+  return selectedPort;
 });
 
 const findManagedPort = (portName: string) => {
-    return app.managedSerialPorts.find(port => port.name === portName) || null;
+  return app.managedSerialPorts.find((port) => port.name === portName) || null;
+};
+
+const isPortOpen = (portName: string): boolean => {
+  return findManagedPort(portName)?.status.type === StatusType.Open;
 };
 
 const limitedPackets = (portName: string) => {
-    const packetLimit = portDisplayPacketsLimits.value[portName] || 100;
```

Comment:
die Pakete waren extra auf 100 limitiert, damit die UI nicht hängt. Diese könnte man später in AppConfig packen.